stories/components/FsForm.vue

```diff
@@ -1,7 +1,7 @@
 <template lang="pug">
-FForm(
+FForm.FForm(
```

Comment:
la classe ici me semble redondante et inutile
```suggestion
FForm(
```